src/libraries/System.Threading.AccessControl/src/System.Threading.AccessControl.csproj

```diff
@@ -21,17 +21,15 @@ System.Security.AccessControl.SemaphoreSecurity</PackageDescription>
     <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">true</IsPartialFacadeAssembly>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(IsPartialFacadeAssembly)' != 'true' and '$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
This is technically a change as we were compiling this file before on the netfx asset. That said it was obviously a mistake so it was a good catch to find here

Reply:
https://github.com/dotnet/runtime/pull/58558#discussion_r701198124